spec/classes/docker_windows_spec.rb

```diff
@@ -28,7 +28,6 @@
       it { should contain_exec('install-docker-package').with_command("& #{service_install_file}") }
       it { should contain_class('docker::repos').that_comes_before('Class[docker::install]') }
       it { should contain_class('docker::install').that_comes_before('Class[docker::config]') }
-      it { should contain_class('docker::service').that_subscribes_to('Class[docker::config]') }
```

Comment:
Can we add in as part of this ensuring that the config class comes before the service class?